charts/influxdb-enterprise/templates/data-configmap.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "influxdb-enterprise.fullname" . }}-data
+  labels:
+    app.kubernetes.io/component: data
+    {{- include "influxdb-enterprise.labels" . | nindent 4 }}
+data:
+  influxdb.conf: |+
+    bind-address = ":8088"
+    reporting-disabled = false
+
+    {{ if .Values.bootstrap.auth.secretName }}
+    [http]
+    auth-enabled = true
+    {{ end }}
+
+    [enterprise]
+      {{ if .Values.license.key }}
+      # license-key and license-path are mutually exclusive, use only one and leave the other blank
```

Comment:
License-key and license-path are mutually exclusive. Use only one and leave the other blank.